identity-and-access-management/audit-trail/how-to/use-audit-trail.mdx

```diff
@@ -0,0 +1,30 @@
+---
+meta:
+  title: How to use Audit Trail
+  description: Learn to view and filter events in Scaleway's Audit Trail by region, date, project, method, and status for effective resource tracking.
+content:
+  h1: How to use Audit Trail
+  paragraph: Learn to view and filter events in Scaleway's Audit Trail by region, date, project, method, and status for effective resource tracking.
+tags: audit-trail events track filter
+dates:
+  validation: 2024-11-26
+  posted: 2024-11-26
+categories:
+  - identity-and-access-management
+---
+
+This page shows you how to see and filter events for your Scaleway resources in Audit Trail.
+
+## How to use Audit Trail
+
+1. Click **Audit Trail** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+2. Click the **Region** dop-down and select a region in which you have created resources integrated with Audit Trail. Your Audit Trail events should display.
+3. Click the **Date range** field and select a date range to track events for.
+4. Click the **Project** drop-down and select the Scaleway Project containing the resources you want to track.
+5. Click the **Method** drop-down and select the API method you want to track events for.
+6. Click the **Status** drop-down and select either **Successful** or **Failed**.
```

Comment:
Let's wait for squad agreement between "Success/fail" or http code, and maybe change the line if needed